### PR TITLE
chore(new-Tests): Adding Pricing Modal testcase

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**

### CODE REVIEW

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-1.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## Change Test Title
+
+---
+
+**Step 1**
+
+Pre-condition:
+
+- should not have been on trial before
+- should currently be on a professional/entriprise license
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-1.md:44]: This precondtion should seems like it should say "on cloud starter" license. I don't see the upgrade button if I upgrade 
 a workspace to cloud professional.

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-1.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+2: Verify 'Upgrade' button is visible when Channel is visited.
+3: Click on the button to verify pricing modal is opened
+
+**Expected**
+
+On pricing modal
+
+1: On starter pack, 'Downgrade' button should be visible and disabled
+2: on professional pack, 'Upgrade' button should be visible and not disabled
+3: on entriprice pack, 'Try free**' button should be visible and disabled
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-1.md:58]: Nit: for all tests, replace `entriprice` with `enterprise` and `Entriprice` with `Enterprise`.

Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-1.md:58]: I think try free should be enabled. Can we remove the two asterisks from try free? I think the quoted text around it is sufficient.

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-2.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## Change Test Title
+
+---
+
+**Step 1**
+
+Pre-condition:
+
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-2.md:42]: There is no professional trial, only enterprise trial. Do you mean requested enterprise trial, and the trial expired and the user was downgraded to starter?

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-3.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Change Test Title
+
+---
+
+**Step 1**
+
+Pre-condition:
+
+- should have requested a professional trial before
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-3.md:43]: The pre-condition should match the description of the test name. I *think* what is desired is that a user has downgraded to cloud starter.

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-3.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+# (Required) Ensure all values are filled up
+name: "Cloud | With no previous trial and no current license Upgrade button for admin users on non trial licensed server"
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-3.md:3]: This sounds like "installation is pre-trial on Cloud Starter" to me.

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-4.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## Change Test Title
+
+---
+
+**Step 1**
+
+Pre-condition:
+
+- should have requested a professional trial before
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-4.md:43]: Update pre-condition.

File: data/test-cases/cloud/pricing-modal/MM-Temp-test-4.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+1: Verify 'Upgrade' button is visible when Channel is visited.
+2: Logout and login with user accout
+3: Verify 'Upgrade' button is not visible
+
+**Expected**
+
+1: Verify pricing modal opens up.
+2: Expect Entriprise section should have 'contact sales' button.
+3: Expect 'Upgrade' button should not be visible.
```
Comment (neallred) [inline data/test-cases/cloud/pricing-modal/MM-Temp-test-4.md:56]: Tests 4-6 are exact duplicates of each other except for the name. Are they all needed? Or maybe some of the test description needs to be updated in them?

**Review by neallred — CHANGES_REQUESTED** [review]

**Comment by yasserfaraazkhan** [comment]
These have been outdated. Will restart again in near future.

**Pull request closed without merging**